haya-hayaodeh/Homework-1/99-bottles-of-beer/app.js

```diff
@@ -10,3 +10,11 @@
   conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else). 
 */
 
+for (let i = 99; i>0 ; i--) {
+  
+ console.log(i + " bottles of beer on the wall, " + i + " bottles of beer.");
+ console.log("Take one down and pass it around, "+ (i-1) + " bottles of beer on the wall.")
+console.log (" ")
```

Comment:
Nice touch to include a blank line between each verse! 
Check out the `\n` statement for the future as well.